packages/bugsnag_flutter/android/src/main/java/com/bugsnag/flutter/BugsnagFlutter.java

```diff
@@ -403,4 +403,21 @@ JSONObject deliverEvent(@Nullable JSONObject eventJson) {
         client.deliverEvent(event);
         return null;
     }
+
+    @Nullable String getString(JSONObject args, String key) {
+        if (!args.has(key)) {
+            return null;
+        }
```

Comment:
I think this check is redundant, if `args.get(key)` returns `null` the `instanceof String` check will fail and we return `null`.